### PR TITLE
Server to send data to

### PR DESCRIPTION
**Pull request by ExcelE**

### CODE REVIEW

File: frontend/app/src/App.js [hunk 1]
```diff
@@ -44,21 +44,6 @@ class App extends Component {
     this.props.history.push("/login");
 
   }
-  handleProfile(){
```
Comment (baivabpokhrel) [inline frontend/app/src/App.js]: we need these for profile routing

File: frontend/app/package.json [hunk 2]
```diff
@@ -6,25 +6,21 @@
     "aws-amplify": "^1.2.4",
     "axios": "^0.19.0",
     "bootstrap": "^4.3.1",
-    "enzyme": "^3.10.0",
-    "jest": "^24.9.0",
```
Comment (baivabpokhrel) [inline frontend/app/package.json]: we need these dependies for testing

File: frontend/app/src/App.js [hunk 3]
```diff
@@ -84,7 +69,7 @@ class App extends Component {
                 ? <Nav pullRight> 
                     
                     <NavItem href="https://foxdot.org/docs/" target="_blank">Documentation</NavItem>
-                    <NavItem onClick={this.handleProfile.bind(this)}>Profile</NavItem>
```
Comment (baivabpokhrel) [inline frontend/app/src/App.js]: we need this for profile routing

File: frontend/app/src/App.test.js [hunk 4]
```diff
@@ -1,14 +1,9 @@
...
 import ReactDOM from 'react-dom';
-
 import App from './App';
 
-
-
-
 it('renders without crashing', () => {
   const div = document.createElement('div');
-  ReactDOM.render(<App.WrappedComponent />, div);
```
Comment (baivabpokhrel) [inline frontend/app/src/App.test.js]: we need this for testing App component as it is exported as a router

**Comment by ExcelE** [comment]
@baivabpokhrel thanks for notifying about the merging issues. I attempted another one. Please look over and see if there's an issue that needs to be resolved.

**Pull request closed without merging**